plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -306,6 +315,10 @@
                 choices:
                     - ReadOnly
                     - ReadWrite
+                default: ReadOnly
+            disk_encryption_set_id:
```

Comment:
Missing type and argument 'disk_encryption_set_id' found in data_disks is listed in DOCUMENTATION.options, but not accepted by the module argument_spec

Reply:
Thank you. I have addressed this. New commit has been pushed.